src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -595,6 +597,20 @@ public void verifyNoAdWithoutTrigger(final String slotSelector) {
   }
 
   public AdsBaseObject triggerAdSlot(String slotName) {
+    if (slotName.equals(AdsContent.BOTTOM_LB) && driver.isChromeMobile()) {
+      waitForSlotExpanded(By.id(AdsContent.MOBILE_AD_IN_CONTENT));
+
+      try {
+        Thread.sleep(1000);
+      } catch (InterruptedException e) {
+        e.printStackTrace();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)

Reply:
Yeah, please log the exception :)